apps/meteor/tests/e2e/e2e-encryption.spec.ts

```diff
@@ -425,6 +425,147 @@ test.describe.serial('e2e-encryption', () => {
 		await expect(poHomeChannel.content.nthMessage(0).locator('.rcx-icon--name-key')).toBeVisible();
 	});
 
+	test.describe('File Encryption', async () => {
+		test.afterAll(async ({ api }) => {
+			expect((await api.post('/settings/FileUpload_MediaTypeWhiteList', { value: '' })).status()).toBe(200);
+			expect((await api.post('/settings/FileUpload_MediaTypeBlackList', { value: 'image/svg+xml' })).status()).toBe(200);
+		});
+
+		test('File and description encryption', async ({ page }) => {
+			await test.step('create an encrypted channel', async () => {
+				const channelName = faker.string.uuid();
+
+				await poHomeChannel.sidenav.openNewByLabel('Channel');
+				await poHomeChannel.sidenav.inputChannelName.fill(channelName);
+				await poHomeChannel.sidenav.advancedSettingsAccordion.click();
+				await poHomeChannel.sidenav.checkboxEncryption.click();
+				await poHomeChannel.sidenav.btnCreate.click();
+
+				await expect(page).toHaveURL(`/group/${channelName}`);
+
+				await poHomeChannel.dismissToast();
+
+				await expect(poHomeChannel.content.encryptedRoomHeaderIcon).toBeVisible();
+			});
+
+			await test.step('send a file in channel', async () => {
+				await poHomeChannel.content.dragAndDropTxtFile();
+				await poHomeChannel.content.descriptionInput.fill('any_description');
+				await poHomeChannel.content.fileNameInput.fill('any_file1.txt');
+				await poHomeChannel.content.btnModalConfirm.click();
+
+				await expect(poHomeChannel.content.lastUserMessage.locator('.rcx-icon--name-key')).toBeVisible();
+				await expect(poHomeChannel.content.getFileDescription).toHaveText('any_description');
+				await expect(poHomeChannel.content.lastMessageFileName).toContainText('any_file1.txt');
+			});
+		});
+
+		test('File encryption with whitelisted and blacklisted media types', async ({ page, api }) => {
+			await test.step('create an encrypted channel', async () => {
```

Comment:
```suggestion
			await test.step('create an encrypted room', async () => {
```
I believe it's not worth it to have distinct tests for groups and channels, so we couldhave a broader description here

Reply:
I understand your concern here, but this is just a naming convention, channels are public rooms, we can't have encrypted "channels" as encrypted rooms can only be private groups.

Also, in code we refer to them everywhere as encrypted channels, so 🤷